docs/community/contributing_guide.mdx

```diff
@@ -60,8 +60,8 @@ own ideas, and experiences.
 
 Other than pull requests and GitHub issues, the best way to contact us is to
 join us at the [CNCF Slack][cncf-slack] under the [**#oras** channel][slack].
-You can find the steps in the [community
-resources](/docs/community/community_resources#joining-the-slack-channel).
+You can find the steps in the
+[community resources](/community/community_resources.mdx#joining-the-slack-channel).
```

Comment:
Should it be
```suggestion
[community resources](/next/community/community_resources/#joining-the-slack-channel).
```

Reply:
This one works
```
* mdx [community resources](/community/community_resources.mdx#joining-the-slack-channel).
```

Reply:
Minimized the changes to just fixing the links

Reply:
WellI am not a frontend expert but `mdx` sounds like a thing that only exist before building out the web page, that's why I am commenting to get it removed from the link. 

Just FYI, I tried `/community/community_resources/#joining-the-slack-channel` and it also works. I am okay with both choices.